presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1594,9 +1594,17 @@ private Scope computeAndAssignOrderByScopeWithAggregation(OrderBy node, Scope ou
                     .collect(toImmutableList());
 
             // generate placeholder fields
+            Set<Field> seen = new HashSet<>();
```

Comment:
do we really care about duplicates? I guess not. Unaliasing will take care about duplicates later.